barretenberg/cpp/src/barretenberg/vm/aztec_constants.hpp

```diff
@@ -85,11 +84,8 @@
 #define AVM_VERSION_BASE_L2_GAS 200
 #define AVM_BLOCKNUMBER_BASE_L2_GAS 200
 #define AVM_TIMESTAMP_BASE_L2_GAS 200
-#define AVM_COINBASE_BASE_L2_GAS 200
```

Comment:
I think you need to remove these from constants.nr as well?

Reply:
huh.... thought i did!

Reply:
oh i have unstaged files....